toolflow/bin/afihelper

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+"""
+Helper script for creating AFIs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This script allows the user to upload a given .tar-file to an Amazon S3
+bucket and to create an AFI (Amazon FPGA Image) from that file.
+
+The AWS environment must be configured, for example by using the AWS CLI
+and running `aws configure`, where the required credentials can be entered.
+
+:Copyright: 2019 Micha Ober
```

Comment:
Please change the copyright header to the proper LGPL one. https://www.gnu.org/licenses/gpl-howto.html